### ISSUE TITLE
include statements not indented correctly?

### ISSUE THREAD

**Issue by endrebak** [kyleam/snakemake-mode#16]
When I write

```
include:
include_prefix + "/rules/window_counts/csaw_window_counts.rules"
```

the region is not indented properly, not even when I mark the region and try to indent it. Glitch? Was hoping for the following, like in the official workflow repo:

```
include:
    include_prefix + "/bio/ngs/rules/mapping/peanut.rules"
```

**Comment by kyleam** [comment]
> Glitch?

Yes, thanks.  I'll add this to the list of things to address when I
rework the indentation functions.

**Comment by kyleam** [comment]
> when I rework the indentation functions

... which I'm doing as part of the changes mentioned [here](https://github.com/kyleam/snakemake-mode/issues/14#issuecomment-213530196).

**Comment by endrebak** [comment]
(Just adding other indentation quirks in this thread)

When there is an empty line between e.g. `input:` and the next, the line is not properly indented.

```
input:

expand(...
```

This is a borderline case, I know.

**Comment by kyleam** [comment]
These cases should be fixed by 1e404ec.  Let me know if you run into
any issues with the new indentation implementation.
